### PR TITLE
Include a `CITATION.cff` file

### PR DESCRIPTION
**Pull request by eddiebergman**
Example output:
```
@proceedings{Feurer_Efficient_and_Robust_2015,
    author = {Feurer, Matthias and Eggensperger, Katharina and Jost, Springenberg and Manuel, Blum and Hutter, Frank},
    booktitle = {Advances in Nerual Infromation Processing Systems 28 (2015)},
    publisher = {Curran Associates, Inc.},
    title = {{Efficient and Robust Automated Machine Learning}},
    url = {https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf},
    year = {2015}
}
```

Preferred citaiton part of the `CITATION.cff`:
```yaml
preferred-citation:
  type: "proceedings"
  title: "Efficient and Robust Automated Machine Learning"
  collection-title: "Advances in Nerual Infromation Processing Systems 28 (2015)"
  year: 2015
  pages: "2962--2970"
  url: "https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf"

  publisher:
    name: "Curran Associates, Inc."

  editors:

    - family-names: "Cortes"
      given-names: "C."

    - family-names: "Lawrence"
      given-names: "N."

    - family-names: "Lee"
      given-names: "D."

    - family-names: "Sugiyama"
      given-names: "M."

    - family-names: "Garnett"
      given-names: "R."

  authors:

    - family-names: "Feurer"
      given-names: "Matthias" 
      orcid: "https://orcid.org/0000-0001-9611-8588"
      affiliation: "University of Freiburg, Germany"

    - family-names: "Eggensperger"
      given-names: "Katharina" 
      orcid: "https://orcid.org/0000-0002-0309-401X"
      affiliation: "University of Freiburg, Germany"

    - family-names: "Jost"
      given-names: "Springenberg"
      orcid: "https://orcid.org/0000-0002-9392-1006"
      affiliation: "University of Freiburg, Germany"

    - family-names: "Manuel"
      given-names: "Blum"
      affiliation: "University of Freiburg, Germany"

    - family-names: "Hutter"
      given-names: "Frank"
      affiliation: "University of Freiburg, Germany"
...
```

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    given-names: "Frank"
+    affiliation: "University of Freiburg, Germany"
+
+preferred-citation:
+  type: "proceedings"
+  title: "Efficient and Robust Automated Machine Learning"
+  collection-title: "Advances in Nerual Infromation Processing Systems 28 (2015)"
+  year: 2015
+  pages: "2962--2970"
+  url: "https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf"
```
Comment (mfeurer) [inline CITATION.cff:51]:
```suggestion
  url: "https://papers.neurips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+    - family-names: "Sugiyama"
+      given-names: "M."
+
+    - family-names: "Garnett"
+      given-names: "R."
+
+  authors:
+
+    - family-names: "Feurer"
```
Comment (mfeurer) [inline CITATION.cff:75]: Why do we have the authors twice?

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  - "machine learning"
+
+license: "BSD-3-Clause"
+
+authors:
+  - family-names: "Feurer"
+    given-names: "Matthias"
+    orcid: "https://orcid.org/0000-0001-9611-8588"
+    affiliation: "University of Freiburg, Germany"
+
```
Comment (mfeurer) [inline CITATION.cff:26]:
```suggestion

  - family-names: "Klein"
    given-names: "Aaron"
    affiliation: "University of Freiburg, Germany"
    
```

File: README.md [hunk 4]
```diff
@@ -33,11 +33,11 @@ Advances in Neural Information Processing Systems 28 (2015)
 [Link](https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf) to publication.
```
Comment (mfeurer) [inline README.md:33]:
```suggestion
[Link](https://papers.neurips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf) to publication.
```

File: CITATION.cff (hunk 2 above)
Comment (eddiebergman) [reply CITATION.cff:79]: One is for the software citation, the first one, the `preferred citation` section refers to the paper and it's the one produced by github.

**Review by mfeurer — APPROVED** [review]

**Pull request merged**